iconrpcserver/utils/json_rpc.py

```diff
@@ -52,14 +52,15 @@ def process_response(self, response, log_extra=None, log_format=None):
             # Log the response before processing it
             self.log_response(response, log_extra, log_format)
             # If it's a json string, parse to object
-            if isinstance(response, basestring):
+            if isinstance(response, str) or isinstance(response, bytes):
                 try:
                     response = json.loads(response)
                 except ValueError:
-                    raise exceptions.ParseResponseError()
+                    # FIXME: Tried to ReceivedErrorResponseError, but needs jsonrpcclient.response.ErrorResponse
+                    raise exceptions.JsonRpcClientError()
```

Comment:
Is there any reason why `ParseResponseError` is replaced with `JsonRpcClientError`?

Reply:
`ParseResponseError` is deprecated in https://github.com/bcb/jsonrpcclient/commit/9638aa18efc84db8fed8c7ca8250150843149cf3

So i choose parent class of the exception, `JsonRpcClientError`. But I'm not sure this exception is acceptable in this line.

Reply:
How about using `ReceivedErrorResponseError`?

Reply:
First time, I used `ReceivedErrorResponseError`, but this exception requires `ErrorResponse`.

Possible `str` or `bytes` response in this line.

Reply:
Thank you for your precious review.

We can discuss the replacement of `CustomAiohttpClient` later, in order to follow bare API of jsonrpcclient.